### PR TITLE
Feat/area coordinates

### LINKED ISSUE #152: Set area coordinates

### PR DESCRIPTION
**Pull request by mheggelund**
closes #152

### CODE REVIEW

File: src/components/AreaCoordinates/AreaCoordinates.tsx [hunk 1]
```diff
@@ -1,161 +1,485 @@
...
+        x: 0,
+        y: 0,
+        m: 0,
+      },
+      {
+        x: 0,
+        y: 0,
+        m: 1,
+      },
+    ],
```
Comment (Sinrefvol) [inline src/components/AreaCoordinates/AreaCoordinates.tsx:80]: Thats the same as defaultArea. Fix?

File: src/components/AreaCoordinates/AreaCoordinates.tsx [hunk 2]
```diff
@@ -1,161 +1,485 @@
...
-  const areaCoordinats = {
-    area: '',
-    topX: '',
-    topY: '',
-    bottomX: '',
-    bottomY: '',
-  };
+  function isNumber(value: any) {
+    return isNaN(value);
+  }
```
Comment (Sinrefvol) [inline src/components/AreaCoordinates/AreaCoordinates.tsx:89]: How does that work?

File: src/components/AreaCoordinates/AreaCoordinates.tsx [hunk 3]
```diff
@@ -1,161 +1,485 @@
...
+            y: 0,
+            m: 0,
+          },
+          {
+            x: 0,
+            y: 0,
+            m: 1,
+          },
+        ],
+      });
```
Comment (Sinrefvol) [inline src/components/AreaCoordinates/AreaCoordinates.tsx:159]: Isnt that equal to defaultArea?

File: src/components/AreaCoordinates/AreaCoordinates.tsx [hunk 4]
```diff
@@ -1,161 +1,485 @@
...
+              m: 0,
+            },
+            {
+              x: 0,
+              y: 0,
+              m: 1,
+            },
+          ],
+        });
+      }
```
Comment (Sinrefvol) [inline src/components/AreaCoordinates/AreaCoordinates.tsx:195]: Same question as before

File: src/components/AreaCoordinates/AreaCoordinates.tsx [hunk 5]
```diff
@@ -1,161 +1,485 @@
...
+          {
+            x: selectedArea[0].coordinates[0].x,
+            y: selectedArea[0].coordinates[0].y,
+            m: 0,
+          },
+          {
+            x: selectedArea[0].coordinates[1].x,
+            y: selectedArea[0].coordinates[1].y,
+            m: 1,
+          },
```
Comment (Sinrefvol) [inline src/components/AreaCoordinates/AreaCoordinates.tsx:216]: How do you know that the list you get from backend is sorted?

File: src/components/AreaCoordinates/AreaCoordinates.tsx [hunk 6]
```diff
@@ -1,161 +1,485 @@
...
+      setNewArea(undefined);
+      setActiveArea(changes.selectedItems[0]);
+      setAreaCoordinate(defaultState);
+    }
+  };
+
+  const validateCoordinates = (area: AreaCoordinateType | undefined) => {
+    const errors: ErrorType = {};
+    if (!activeArea || activeArea.modelAreaTypeId === '') {
+      errors.area = 'Moedl area needs to be selected';
```
Comment (Sinrefvol) [inline src/components/AreaCoordinates/AreaCoordinates.tsx:232]: typo: model

File: src/components/AreaCoordinates/AreaCoordinates.tsx [hunk 7]
```diff
@@ -1,161 +1,485 @@
...
+          y: 0,
+          m: 0,
+        },
+        {
+          x: 0,
+          y: 0,
+          m: 1,
+        },
+      ],
+    });
```
Comment (Sinrefvol) [inline src/components/AreaCoordinates/AreaCoordinates.tsx:317]: Isnt that a defaultArea?

**Review by Sinrefvol — CHANGES_REQUESTED** [review]

File: src/components/AreaCoordinates/AreaCoordinates.tsx (hunk 5 above)
Comment (mheggelund) [reply src/components/AreaCoordinates/AreaCoordinates.tsx:216]: fixed

File: src/components/AreaCoordinates/AreaCoordinates.tsx (hunk 6 above)
Comment (mheggelund) [reply src/components/AreaCoordinates/AreaCoordinates.tsx:232]: fixed

File: src/components/AreaCoordinates/AreaCoordinates.tsx (hunk 2 above)
Comment (mheggelund) [reply src/components/AreaCoordinates/AreaCoordinates.tsx:89]: Check if value is a number, returns a boolean. 

Built in JS method

**Pull request merged**